document-store/src/main/java/org/hypertrace/core/documentstore/model/config/ConnectionConfig.java

```diff
@@ -38,11 +40,11 @@ public static ConnectionConfigBuilder builder() {
   @FieldDefaults(level = AccessLevel.PRIVATE)
   public static class ConnectionConfigBuilder {
     DatabaseType type;
-    String host = DEFAULT_HOST;
-    Integer port;
+    List<Endpoint> endpoints = new ArrayList<>();
     String database;
     ConnectionCredentials credentials;
     String applicationName = DEFAULT_APP_NAME;
+    String replicaSet;
```

Comment:
- How was this pass in earlier version where we used to pass only TypeSafe Config?
- Was it not required prior?

Reply:
Yeah. Not required earlier.